HotelBooking/Views/pay_bill_view.py

```diff
@@ -101,21 +107,17 @@ def pay_bill(self):
                 bill_id = answer_list[1].strip()
                 self.bill_controller.pay_bill(bill_id)
                 print("\nSuccess!\n")
-        self.show()
+                self.show_again()
+            
+        self.prev_view()
+        
 
     def list_bill(self):
```

Comment:
should be `list_bills` since you're printing all bills.

Reply:
petty af

Reply:
also Vincent named it that but Ill change it lol